tests/test_selector_csstranslator.py

```diff
@@ -236,3 +236,9 @@ def test_nested_selector(self) -> None:
     )
     def test_pseudoclass_has(self) -> None:
         self.assertEqual(self.x("p:has(b)::text"), ["lorem ipsum text"])
+
+
+
+class CSSSelectorTestBytes(CSSSelectorTest):
+    def setUp(self):
+        self.sel = self.sscls(bytes_input=bytes(HTMLBODY, encoding="utf8"))
```

Comment:
This seems to be causing test failures.

```suggestion
        self.sel = self.sscls(body=bytes(HTMLBODY, encoding="utf8"))
```

Reply:
Fixed